packages/ui/src/ui/components/Dialog/DialogError.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import cn from 'bem-cn-lite';
+
+import type {FORM_ERROR as FormErrorType} from './index';
+
+import {FIX_MY_TYPE, YTError} from '../../types';
+import Block from '../../components/Block/Block';
+
+import compact_ from 'lodash/compact';
+import map_ from 'lodash/map';
+
+const block = cn('yt-dialog-error');
+
+export function DialogError(props: FIX_MY_TYPE) {
+    return <Block {...props} className={block()} />;
+}
+
+export function makeErrorFields(errors: Array<YTError | Error | undefined>) {
```

Comment:
I recommend putting this in a separate file. Because this function is located in the DialogError.tsx component, but it is not actually used in it.